avm/res/compute/virtual-machine/tests/e2e/atmg/main.test.bicep

```diff
@@ -20,6 +20,10 @@ param serviceShort string = 'cvmlinatmg'
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
+// Set to fixed location as the RP function returns unsupported locations (configurationProfileAssignments)
+// Right now (2024/04) the following locations are supported: centralus, eastus, eastus2, southcentralus, westus, westus2, westcentralus, northeurope, westeurope, canadacentral, japaneast, uksouth, australiasoutheast, australiaeast, southeastasia, westus3
+param enforcedLocation string = 'westeurope'
```

Comment:
@rahalan we should not use West Europe. Please change in your next PR

Reply:
@AlexanderSehr Are there any guidelines on what region must be used in this situation?

Reply:
In other Bicep templates the `enforcedLocation` variable is scoped to either `eastus` or `eastasia`.  So eastus would be the choice here. Is that correct?